### ISSUE TITLE
📜 Slides sound "learner facing"

### ISSUE THREAD

**Issue by Felienne** [hedyorg/hedy#5028]
The content on the slides is meant to be used in lessons, but sounds a bit like it is speaking directly to an (individual) student, f.e. here in level 13:

<img width="1205" alt="image" src="https://github.com/hedyorg/hedy/assets/1003685/7c596489-6886-4eee-a8be-30ab4221ce8d">

Let's try to make this a bit more systematic?

**Comment by MarleenGilsing** [comment]
Yes, I think I made this with teachers that don't know that much about programming themselves in mind. Maybe we can discuss this (at the hackathon?). What kind of teachers do we cater and what are their needs? I always kind of assume that they're teacher like me, who don't know much of programming but want to teach it to their kids and need a a lot of context and explanation themselves. But maybe our teacher population exists way more of programming experts, who immediately understand slides without any help and just want them to be concise.

**Comment by Felienne** [comment]
Can be combined with [#5805](https://github.com/hedyorg/hedy/issues/5805)
